tools/SourceKit/lib/SwiftLang/CMakeLists.txt

```diff
@@ -64,3 +64,20 @@ target_link_libraries(SourceKitSwiftLang PRIVATE
   clangAPINotes
   clangBasic)
 add_dependencies(SourceKitSwiftLang clang-tablegen-targets)
+
+if (NOT LLVM_ENABLE_ASSERTIONS)
```

Comment:
I need to propagate this to the rest of the CMake changes below. But I think in principle this works for Linux/macOS.